float32 dist
---
bool status
